### DISCUSSION TITLE
Does useSuspenseQuery always try api requests on the server as well?

### DISCUSSION THREAD

**Discussion by JhsK** [TanStack/query#7214] [Q&A] [answered]
If useSuspenseQuery + Suspense is used together, Next.js(v13) + page router is sending an Api request at build without using the getStaticProps function when building SSG, can you tell me why this happens?

The image below is the image that the terminal outputs when building after creating console.log in the axios function.
![image](https://github.com/TanStack/query/assets/69228184/6de7e19a-e899-4884-898a-6329eaf88ce5)

**Comment by TkDodo** [comment, marked as answer]
suspense is not supported in pages router on the server afaik because it doesn't support streaming. I wouldn't know why a request gets made ...

**Reply by JhsK** [reply]
@TkDodo 
Oh I see! From what you answered, I realized that if I don't use the server component of the app router, I can't use Suspense for SSR on the page router!

If so, if I use Suspense on the page router, is it right to implement it using the ssr:false method of dynamic import or clientOnly of `@suspensive/suspense`?

**Reply by TkDodo** [reply]
yep

**Reply by JhsK** [reply]
@TkDodo Thank you so much for your response!
